Packs/CVE_2021_44228/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+##### CVE-2021-44228 - Log4j RCE
+- On Dec 18 2021, another vulnerability was discovered related the log4j 0-day exploit known as CVE-2021-45105
```

Comment:
- On Dec 18 2021, another vulnerability was discovered related the log4j 0-day exploit known as CVE-2021-45105.
- Added a new threat ID. 
- Added the new patch for mitigation.
- Playbook will also search for potentially vulnerable servers using Xpanse.